src/sections/General/Footer/footer.style.js

```diff
@@ -232,6 +232,30 @@ const FooterWrapper = styled.section`
 			padding: 0 1rem;
 		}
 	}
+	
+	
+	@media screen and (max-width: 445px) {
+           .terms{
+              position: relative;
+              top: -32px;
+              left: 30px;
+            }
+           }
+
+        @media screen and (max-width: 62rem) {
+           .privacy{
+               position: relative;
+               left: -28px;
+             }
+           }
+
+        @media screen and (max-width: 62rem) {
+           .status{
+              position: relative;
+              left: -28px;
+            }
+          }
```

Comment:
Could be aggregated under one media query.
Also, for media queries it is better to use `px` unit for resolutions over `rem`.